src/main/java/crazypants/enderio/conduit/item/filter/ExistingItemFilter.java

```diff
@@ -78,7 +78,7 @@ boolean stackEqual(ItemStack toInsert, ItemStack existing) {
       matched = existingId != -1 && existingId == OreDictionary.getOreID(toInsert);
     }
     if(!matched) {
-      matched = Item.getIdFromItem(toInsert.getItem()) == Item.getIdFromItem(existing.getItem());
```

Comment:
I wonder what couldbe the reason for that

Reply:
Fluids can be compared by their IDs only. I guess EnderIO author thought the item would be the same.